docker/README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Available image names
 
-- `ghcr.io/tinkoff-ai/etna/etna-cpu:<tag>`
-- `ghcr.io/tinkoff-ai/etna/etna-cuda-11.6.2:<tag>`
+- `ghcr.io/etna-team/etna/etna-cpu:<tag>`
+- `ghcr.io/etna-team/etna/etna-cuda-11.6.2:<tag>`
 
 On previous versions of the library there were also:
-- `ghcr.io/tinkoff-ai/etna/etna-cuda-10.2:<tag>`
```

Comment:
It's not working at the moment

Reply:
What do you mean? Can you elaborate on this?